portal/celery_worker.py

```diff
@@ -33,11 +36,17 @@ def setup_periodic_tasks(sender, **kwargs):
         test_job = db.session.merge(test_job)
 
     # add all tasks to Celery
-    for job in ScheduledJob.query.filter_by(active=True):
+    logger.info("ScheduledJobs found: {}".format(ScheduledJob.query.count()))
+    for job in ScheduledJob.query.all():
         task = getattr(tasks, job.task, None)
         if task:
+            logger.info("Adding task (id=`{}`, task=`{}` "
+                        "to CeleryBeat".format(job.id, job.task))
             args_in = job.args.split(',') if job.args else []
             kwargs_in = job.kwargs or {}
-            sender.add_periodic_task(job.crontab_schedule(),
-                                     task.s(job.id, *args_in,
-                                            **kwargs_in))
+            kwargs_in['job_id'] = job.id
+            try:
+                sender.add_periodic_task(job.crontab_schedule(),
+                                         task.s(*args_in, **kwargs_in))
+            except Exception as exc:
+                logger.info(exc)
```

Comment:
Shouldn't this write to a different log level (`warn`/`error`)?

Reply:
Good point, I'll update that.